openhtf/output/web_gui/__init__.py

```diff
@@ -331,9 +332,37 @@ def __init__(self, discovery_interval_s, disable_discovery, http_port,
         (r'/', self.MainHandler, {'port': http_port}),
         (r'/station/(?:(?:[0-9]{1,3}\.){3}[0-9]{1,3})/(?:[0-9]{1,5})(?:/*)',
          self.MainHandler, {'port': http_port}),
-        (r'/(.*\..*)', tornado.web.StaticFileHandler, {'path': path}),
-        (r'/(styles\.css)', tornado.web.StaticFileHandler, {'path': path}),
+        (r'/(.*\..*)', tornado.web.StaticFileHandler, {'path': path})
     ] + dash_router.urls + station_router.urls
+
+    #Add plugins into handler_routes
+    if type(plugins) == list:
+      for name in plugins:
+        try:
+          #Import plugin package
+          plugin_module = importlib.import_module(name)
+          handler,deps = plugin_module.handler
+
+          #Add in handler dependencies 
+          dep_dict = {}
+          for dep in deps:
```

Comment:
Why not pass in a `namedtuple` with all of these attributes? Then they only have to supply the handler itself